packages/ui/__tests__/Radio.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { mountAndCheckA11Y } from '@hazelcast/test-helpers'
+
+import { v4 as uuid } from 'uuid'
+import { Radio } from '../src/Radio'
+
+jest.mock('uuid')
+
+const uuidMock = uuid as jest.Mock<ReturnType<typeof uuid>>
+
+describe('Radio', () => {
```

Comment:
Can we add these tests?
1. Radio renders label and and input. (Just to make sure everything expected renders).
2. Radio renders helperText.

Reply:
@jankoritak I'll do that thanks!